src/SignalRServiceExtension/Bindings/LiteServiceEndpoint.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.SignalR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class LiteServiceEndpoint
+    {
+        public string ConnectionString { get; set; }
+
+        public EndpointType EndpointType { get; set; }
+
+        public string Name { get; set; }
+
+        public string Endpoint { get; set; }
+
+        public bool Online { get; set; }
+
+        public static LiteServiceEndpoint FromServiceEndpoint(ServiceEndpoint e)
```

Comment:
Make this internal or private which is not aware by customer? Do same to `ToServiceEndpoint()`.